### PR TITLE
Add Owner permission in pre-reqs

### PR DESCRIPTION
**Pull request by sebader**
Make this even more clear

### CODE REVIEW

File: docs/reference-implementation/Getting-Started.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ This guide describes the end-to-end process for setting up all pre-requisites an
 
 ## Pre-requisites
 
+The Azure Mission-Critical reference implementation gets deployed into an Azure Subscription. For this you need to have either:
+
+- **Owner permissions** yourself and to be able to create a Service Principal which you are able to grant Owner as well, or
+- A pre-provided Service Principal with Owner permissions on a subscription
```
Comment (heoelri) [inline docs/reference-implementation/Getting-Started.md:20]:
```suggestion
- A pre-provisioned Service Principal with Owner permissions on a subscription
```

LGTM - but isn't pre-provisioned the more common terminology?

File: docs/reference-implementation/Getting-Started.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ This guide describes the end-to-end process for setting up all pre-requisites an
 
 ## Pre-requisites
 
+The Azure Mission-Critical reference implementation gets deployed into an Azure Subscription. For this you need to have either:
+
+- **Owner permissions** yourself and to be able to create a Service Principal which you are able to grant Owner as well, or
```
Comment (hjscherer) [inline docs/reference-implementation/Getting-Started.md:19]: Owner permission doesn't allow the user to create an SPN. Can we separate it and we can also mention that User Access Administrator (UAA) would also be enough to make a role assignment for the SPN

**Review by heoelri — APPROVED** [review]

**Review by hjscherer — CHANGES_REQUESTED** [review]

File: docs/reference-implementation/Getting-Started.md (hunk 2 above)
Comment (sebader) [reply docs/reference-implementation/Getting-Started.md:19]: hmm good point. What permissions do you need to create an SPN?

Comment (heoelri) [reply docs/reference-implementation/Getting-Started.md:19]: Aren't these the two viable options we have here:

a) use a pre-provisioned service principals with subscription-level owner permissions 
b) auto-create a spn as part of the creation of a service connection in azure devops (what permissions does that require)?

Option a) implies that you have to get one (or create one yourself) 

We're not global administrators in the microsoft tenant, but we can use the auto create option in ADO, can't we? And we can also manually create SPs.

Comment (hjscherer) [reply docs/reference-implementation/Getting-Started.md:19]: App registrations in the User Settings on the tenant
image

Comment (sebader) [reply docs/reference-implementation/Getting-Started.md]: how about now?

Comment (hjscherer) [reply docs/reference-implementation/Getting-Started.md]: added a minor change

**Review by hjscherer — APPROVED** [review]

**Pull request merged**